src/modules/flow/iio/iio.json

```diff
@@ -421,8 +421,8 @@
            "name": "use_device_default_scale"
          },
          {
-             "data_type": "float",
-             "default": 0,
+             "data_type": "direction-vector",
```

Comment:
If we can use different scale and offset for each R, G or B, we'll need to have three options for each =/
Use direction-vector is convenient, but is also very counter-intuitive.